crates/noirc_evaluator/src/ssa/node.rs

```diff
@@ -521,8 +531,12 @@ impl Instruction {
 #[derive(Debug, PartialEq, Eq, Hash, Clone)]
 pub enum Operation {
     Binary(Binary),
-    Cast(NodeId),                                                 //convert type
-    Truncate { value: NodeId, bit_size: u32, max_bit_size: u32 }, //truncate
+    Cast(NodeId), //convert type
+    Truncate {
+        value: NodeId,
+        bit_size: u32,
+        max_bit_size: u32,
+    }, //truncate
```

Comment:
Looks like `use_small_heuristics = "Max"` in our .rustfmt.toml broke.
Looking at https://rust-lang.github.io/rustfmt I no longer see it as an option so I wondered if it was removed but it doesn't seem to be removed in the rustfmt repo.